### ISSUE TITLE
invalid parameter "pages" - with a single page number as string

### ISSUE THREAD

**Issue by RikaiDesign** [nbesli/pdf-merger-js#87]
add (inputFile, pages) {

processes **pages** as an array or string. 
As a string, it processes the ','  '-' 'to'  but doesn't process a single page number in a string

**Comment by mojoaxel** [comment]
I also thinks we should allow simple numbers and strings like this in the futue. It should be quite simple to implement this. We just need to extend the `add` function. Merge requests are always welcome!

```js
await merger.add(inputFile, 1);
await merger.add(inputFile, '12');
```

**Comment by Furkangunduz** [comment]
Hello can you check my pull request ? #89

**Comment by mojoaxel** [comment]
@Furkangunduz Thanks for fixing this with #89

**Comment by Ocyss** [comment]
I found that if I use a single numeric string as pages, I will still get an error. I used version 4.3 and got an error. I confirmed that there was no problem with my code and tried 4.0.1, but the same error was reported.
`Error: invalid parameter "pages"`

PS: What is printed in the picture below is an array, but I am using a for loop to perform the add operation, so I am passing a single string.

![image](https://github.com/nbesli/pdf-merger-js/assets/68412205/43b9327a-b7d4-48cd-a55c-d12a313d5add)
